src/hooks/useSubaccount.tsx

```diff
@@ -122,7 +129,8 @@ const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: LocalWall
         try {
           return await compositeClient?.withdrawFromSubaccount(
             subaccountClient,
-            amount.toFixed(usdcDecimals)
+            amount.toFixed(usdcDecimals),
+            `${DEFAULT_TRANSACTION_MEMO} | withdraw from subaccount to wallet`
```

Comment:
nit: For the additional memo info maybe we want to use an `enum` or helper method instead of hard coding within `useSubaccount` so we can have a central place for **all** memos